src/renderer/angular/controllers/commons/dialogs/send-token-controller.js

```diff
@@ -92,6 +92,11 @@ function SendTokenDialogController($rootScope, $scope, $log, $q, $mdDialog, $int
                 }
             }
 
+            if(error.indexOf('Endpoint request timed out') !== -1){
```

Comment:
I'd highly suggest not to have hardcoded string checks in code, move to ``constansts`` instead.
When needed, you could change constant instead of searching codebase for ``strings``

Reply:
Just noticed that there are 8 findings of ``error_while_creating`` inside ``src/main/controllers/sql-lite-service.js``.